crypto/subtlecrypto.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crypto
+
+import (
+	"context"
+
+	"github.com/lestrrat-go/jwx/v2/jwk"
+)
+
+// SubtleCrypto offers an interface to perform low-level ("subtle") cryptographic operations with keys stored in a vault.
+type SubtleCrypto interface {
```

Comment:
Can you please add GetComponentMetadata the same way we have in State Store and Secret Store?

Reply:
Done